src/client/components/dataItem/formatLabelFromProps.test.js

```diff
@@ -0,0 +1,26 @@
+import { expect } from 'chai';
+
+import { formatLabelFromProps } from './formatLabelFromProps';
+
+describe('formatLabelFromProps', () => {
+  const mockProps = {
+    label: 'Volume'
+  };
+
+  it('properly formats label from props with no optional label', () => {
+    expect(formatLabelFromProps(mockProps))
+      .to.equal(mockProps.label);
```

Comment:
Often in your tests you create a mock object and then expect a result of `mockObject.some.property` (in this case, `mockProps.label`).  I think this might be bad though, because mockObject might have been modified by the function you just called it with.  I think you should either

a) Object.freeze the mockObject when you create it
or
b) Define the mockObject in terms of mock properties.  Here, that would mean
```javascript
const mockLabel = 'Volume';
const mockProps = { label: mockLabel };

...

 expect(formatLabelFromProps(mockProps))
       .to.equal(mockLabel);
```